### ISSUE TITLE
[dependencies] Update sharp version 

### ISSUE THREAD

**Issue by NinaNeira** [ikhsanalatsary/multer-sharp-s3#21]
sharp version needs to be updated in order to support Node v12. Version in your dependencies causes compilation failure because Node v12 removed deprecated `v8::Handle`.

**Comment by jeffminsungkim** [comment]
@NinaNeira Agree.

**Comment by jeffminsungkim** [comment]
@ikhsanalatsary I've just sent you an email related to this issue. Please give us some feedback. It's quite urgent.

**Comment by jaybuddy** [comment]
Has there been any update on this? Figured I would bump it to see if we can get it updated. Looks like there is already a PR ready to go in the repo.

**Comment by samlevin** [comment]
@ikhsanalatsary has this project been abandoned?

**Comment by DevNM21** [comment]
guys any update?

**Comment by DevNM21** [comment]
or at least any quick-fix anybody knows?

**Comment by NinaNeira** [comment]
I've tried many different solutions, but at the end was forced to downgrade Node version in my project (to 10.16.3).

You can always use other packages or write some Serverless API.

**Comment by brianchirls** [comment]
I was able to get a fix working and made a pull request. It's on my branch [here](https://github.com/TechnologyHumansAndTaste/multer-sharp-s3/tree/sharp-upgrade-0.24.0) if anyone wants to use that in the meantime.

**Comment by DevNM21** [comment]
@brianchirls how do I use your branch in my project?

**Comment by DevNM21** [comment]
@brianchirls  Aw, man. I love you, it works

**Comment by brianchirls** [comment]
@DevNM21 I'm glad it's working for you! Happy to share my fix with the community.

In case anybody's wondering how to use my branch, this should do it:
```bash
npm install github:TechnologyHumansAndTaste/multer-sharp-s3#sharp-upgrade-0.24.0
```

**Comment by brianchirls** [comment]
Ugh, sharp release a new version *yesterday* - I'll look into it further patching this PR/branch.

**Comment by CaptainAwesomeDi** [comment]
@brianchirls I don't know if this project is still actively maintained. 
Do you want to maintain it? put it to a different node package?

**Comment by DevNM21** [comment]
@CaptainAwesomeDi  that is a great thought

**Comment by ikhsanalatsary** [comment]
hi everyone, I already merged #27 into development, I need check that first because some test was failed in Travis ci

**Comment by ikhsanalatsary** [comment]
closed due to #28 & has been release to npm registry with v0.2.0
